SPBypassLoginPage/LOGIN/Bypass/BypassLogin.aspx.cs

```diff
@@ -32,21 +32,27 @@ public static string GetSubString(string value, char separator, int index)
     public partial class BypassLogin : IdentityModelSignInPageBase
     {
         const string CustomLoginProperty = "CustomBypassLogin";
+        const string WindowsAuthIPsConfig = "WindowsAuthIPs";
```

Comment:
The name of the key should be explicitly related to this project. 
What about CustomBypassLogin_WindowsAuthIPs? Or CustomBypassLogin_WindowsClientIP?
Also, I think it supports only a single IP. If so, can you rename it to use singular instead of plural?

Reply:
I'm neutral in this point... 
It supports more than one IP (as described in the first comment)

Reply:
Ok, I renamed property to CustomBypassLogin_WindowsAuthIPs